packages/Python/lldbsuite/test/lang/swift/optional_url/main.swift

```diff
@@ -0,0 +1,4 @@
+import Foundation
+
+let u : URL? = URL(string: "https://github.com")
+print(u) //%self.expect('po u', substrs=['https://github.com'])
```

Comment:
As usual, a test without foundation would be better.

Reply:
Does the test also work with frame variable?

Reply:
No, the formatter is broken for other reasons. But this at least fixes `po`.

Reply:
Could you please file a bug for that, too?

Reply:
There's already a rdar. <rdar://problem/51780502>